ibis/pyspark/compiler.py

```diff
@@ -332,18 +332,21 @@ def compile_aggregator(t, expr, scope, fn, context=None, **kwargs):
         src_col = F.when(condition, src_col)
 
     col = fn(src_col)
-    if context is None:
+    if (
+        context == AggregationContext.ENTIRE
+        or context == AggregationContext.GROUP
+    ):
+        return col
+    elif context == AggregationContext.WINDOW:
+        window = kwargs['window']
+        return col.over(window)
+    else:
```

Comment:
This is just reorganizing the if-else block for better readability

Making this change here is kind of orthogonal, but I use the same logic in the compilation function for `ReductionVectorizedUDF` and I'd like the logic in both places to look similar